### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by JayeshVP24**
## Screenshots (if appropriate)

### CODE REVIEW

File: src/pages/api/listings/scout/[id].ts [hunk 1]
```diff
@@ -110,17 +109,17 @@ async function scoutTalent(req: NextApiRequestWithUser, res: NextApiResponse) {
       },
     });
 
-    if (prevScouts[0]) {
```
Comment (a20hek) [inline src/pages/api/listings/scout/[id].ts:113]: what was this condition for?

File: src/components/home/SideBar.tsx [hunk 2]
```diff
@@ -320,9 +320,11 @@ export const HomeSideBar = ({
           <LiveListings />
         </>
       )}
-      {router.asPath === '/' && status === 'unauthenticated' && !session && (
-        <SponsorBanner />
-      )}
+      {router.asPath === '/' &&
```
[the remark this answers is not in the document]
Comment (JayeshVP24) [reply src/components/home/SideBar.tsx:323]: need to show the sponsor banner to logged in but no profile (neither talent nor sponsor profile) created users as well

File: src/features/comments/components/CommentParser.tsx [hunk 3]
```diff
@@ -12,11 +12,19 @@ interface Props {
   submissionId?: string;
 }
 
+export const isLink = (text: string) => {
```
[the remark this answers is not in the document]
Comment (JayeshVP24) [reply src/features/comments/components/CommentParser.tsx:15]: submission details page in sponsor dashboard. should i make this shared util?

File: src/features/listings/components/Submission/SubmissionModal.tsx [hunk 4]
```diff
@@ -203,7 +203,7 @@ export const SubmissionModal = ({
       }
 
       const latestSubmissionNumber = (userInfo?.Submission?.length ?? 0) + 1;
-      if (!editMode) showEasterEgg();
+      if (!editMode && latestSubmissionNumber === 1) showEasterEgg();
```
[the remark this answers is not in the document]
Comment (JayeshVP24) [reply src/features/listings/components/Submission/SubmissionModal.tsx:206]: To only show the easter egg once on first submission
pratik asked to do this

File: src/pages/api/listings/scout/[id].ts (hunk 1 above)
Comment (JayeshVP24) [reply src/pages/api/listings/scout/[id].ts:113]: this was to be reverted after prathiks test, will do it

Comment (JayeshVP24) [reply src/pages/api/listings/scout/[id].ts]: reverted

File: src/features/comments/components/CommentParser.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (a20hek) [reply src/features/comments/components/CommentParser.tsx:15]: yup

File: src/features/listings/components/Submission/SubmissionModal.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (a20hek) [reply src/features/listings/components/Submission/SubmissionModal.tsx:206]: okay

**Pull request merged**